README.md

```diff
@@ -1124,6 +1128,135 @@ Available aggregations are:
 - geometry_geohash_grid_frequency ([geohash grid](https://opensearch.org/docs/latest/aggregations/bucket/geohash-grid/) on Item.geometry)
 - geometry_geotile_grid_frequency ([geotile grid](https://opensearch.org/docs/latest/aggregations/bucket/geotile-grid/) on Item.geometry)
 
+## Asset Proxy
+
+The Asset Proxy feature enables stac-server to proxy access to S3 assets through the STAC
+API by generating requester-pays pre-signed URLs. Only assets with S3 URIs (`s3://`
```

Comment:
This suggest that this is for request-pays buckets only, but this should work regardless of how the bucket is configured.